docs/usage/self-hosted-configuration.md

```diff
@@ -92,6 +92,12 @@ If you configure this to be different to the `baseDir`, it means you can have on
 
 Set to `false` to prevent usage of `--ignore-platform-reqs` in the Composer package manager.
 
+## dockerChildPrefix
+
+Override the default renovate sidecar Docker containers name prefix from `renovate_` to a custom value, so renovate will name and label images from a custom prefix.
+
+If this is set to `myprefix_` the final image name for `renovate/node` would be named `myprefix_renovate_node` instead of currently used `renovate_node` and be labeled `myprefix_child` instead of `renovate_child`.
```

Comment:
Is it really `renovate_node` by default or `renovate_renovate_node` ?

Reply:
Previously we have not added any prefix to the container name so `renovate/node` would have been named `renovate_node` as container name and received `renovate_child` as label.

Now we prefix all container names with `renovate_` by default so `renovate/node` becomes `renovate_renovate_node` and label all container with `<prefix>child` so it becomes `renovate_child` as before.

Reply:
But yes, the documentation refers to the old style, I will adjust it.